src/opencl_common.c

```diff
@@ -245,8 +250,8 @@ void opencl_driver_value(int sequential_id, int *major, int *minor)
 
 	*major = 0, *minor = 0;
 
-	clGetDeviceInfo(devices[sequential_id], CL_DRIVER_VERSION,
-	                sizeof(dname), dname, NULL);
+	HANDLE_CLERROR(clGetDeviceInfo(devices[sequential_id], CL_DRIVER_VERSION,
```

Comment:
It doesn't abort from 0.0 or any other value in `dname`, it only aborts if `clGetDeviceInfo()` returns an error (eg. `CL_INVALID_DEVICE`). Otherwise we may chug along and fail later with less of a clue.